_episodes/43.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: episode
+title: "43: Trino saves trips with Alluxio"
+date: 2022-12-15
+tags: cache alluxio orchestration storage file
+youtube_id: "---Jp9d-1qQUH0---"
+wistia_id: ""
+sections:
+   - title: "Concept of the episode"
+     desc: "Data caching and orchestration"
+     time: 0
+   - title: "Concept of the episode"
+     desc: "Trino without caching"
+     time: 0
+   - title: "Concept of the episode"
+     desc: "Hive connector caching"
+     time: 0
+   - title: "Concept of the episode"
+     desc: "What is Alluxio?"
+     time: 0
+   - title: "Concept of the episode"
+     desc: "What is data orchestration?"
+     time: 0
+   - title: "Concept of the episode"
+     desc: "Trino and Alluxio"
+     time: 0
+   - title: "PR of the episode"
+     desc: "Alluxio/alluxio PR 13000 Add a doc for Trino"
+     time: 0
+   - title: "Demo of the episode"
+     desc: "Running Trino on Alluxio"
+     time: 0
+---
+
+## Hosts
+
+* Brian Olsen, Developer Advocate at [Starburst](https://starburst.io)
+  ([@bitsondatadev](https://twitter.com/bitsondatadev))
+* Manfred Moser, Director of Information Engineering at 
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+* Bin Fan, VP of Open Source at Alluxio and PMC maintainer of Alluxio open 
+  source and TSC member of Presto ([@binfan](https://twitter.com/binfan))
+* [Beinan Wang](https://www.linkedin.com/in/beinan/), Software Engineer at 
+  Alluxio and Presto committer
+
+## Concept of the episode: Data caching and orchestration
+
+Out of all those petabytes of data you store, only a small fraction of it is
+creating business value for you today. When you scan the same data multiple
+times and transfer it over the wire, you're wasting time, compute cycles, and
+ultimately money. This gets worse when you're pulling data across regions or
+clouds from disaggregate Trino clusters. In situations like these, caching
+solutions can make a tremendous impact on the latency and cost of your queries.
+
+### Trino without caching
```

Comment:
we need to add that this is about caching for object files.. and therefore only applies to object storage connectors (hive, iceberg, delta lake and hudi)